v0/destinations/mautic/transform.js

```diff
@@ -0,0 +1,236 @@
+const {
+  defaultRequestConfig,
+  CustomError,
+  constructPayload,
+  // isDefinedAndNotNull,
+  // getIntegrationsObj,
+  removeUndefinedAndNullValues,
+  getErrorRespEvents,
+  getSuccessRespEvents,
+  getDestinationExternalID,
+  defaultPostRequestConfig,
+  defaultPatchRequestConfig
+} = require("../../util");
+
+const {
+  FIELDS,
+  validateEmail,
+  deduceAddressFields,
+  validatePayload,
+  searchContactId,
+  refinePayloadFields
+} = require("./utils");
+
+const { EventType } = require("../../../constants");
+
+const { BASE_URL, mappingConfig, ConfigCategories } = require("./config");
+
+const responseBuilderGroup = async (endpoint, destination) => {
```

Comment:
Please merge `responseBuilderGroup` and `responseBuilderIdentify` and get rid of code duplication.